AndelaOSP/art-dashboard
# #159522791 Logout user when token expires

## What does this PR do?
Logs out the user when their token has expired
## Description of Task to be completed?
When a user is inactive for X amount of time e.g 1 hr, they should be automatically logged out.
## How should this be manually tested?
Pull this branch
After one hour of inactivity on your account, observe a Message informing you that your session has timed out.
## What are the relevant pivotal tracker stories?

## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos

## Screenshots
<img width="1439" alt="screen shot 2018-08-16 at 5 27 43 pm" src="https://user-images.githubusercontent.com/26274319/44221894-d9fb3080-a17a-11e8-97b0-91baafda0ba8.png">
